etcdctl/ctlv3/command/snapshot_command.go

```diff
@@ -50,10 +50,7 @@ func snapshotSaveCommandFunc(cmd *cobra.Command, args []string) {
 		cobrautl.ExitWithError(cobrautl.ExitBadArgs, err)
 	}
 
-	lg, err := logutil.CreateDefaultZapLogger(zap.InfoLevel)
-	if err != nil {
-		cobrautl.ExitWithError(cobrautl.ExitError, err)
-	}
+	lg := logutil.CreateUtilZapLogger(zap.InfoLevel)
```

Comment:
The problem I spot here is that it will only fix the issue for `snapshot save`. I'm curious about what's the conclusion of issue #17464. So, maybe this fix should expand to other commands, not only this one.

Reply:
There's still one instance in which it still calls `CreateDefaultZapLogger`. Refer to `etcdctl/ctlv3/command/global.go:141`.